taskcluster/app_services_taskgraph/__init__.py

```diff
@@ -42,7 +42,7 @@ def get_decision_parameters(graph_config, parameters):
                     head_tag
                 )
             )
-        version = get_version()
+        version = get_version(graph_config)
```

Comment:
Noting that technically this is a `kind_config` not a `graph_config`. But the misnomer is pre-existing so not something you need to fix here.